src/Dropdown.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { Person } from './types/Person';
+
+interface DropdownProps {
+  isFocused: boolean;
+  inputValue: string;
+  showSuggestions: boolean;
+  filteredPeople: Person[];
+  handleInputChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  handleInputFocus: () => void;
+  handleInputBlur: () => void;
+  handleSuggestionClick: (person: Person) => void;
+}
+
+export const Dropdown: React.FC<DropdownProps> = ({
+  isFocused,
+  inputValue,
+  filteredPeople,
+  showSuggestions,
+  handleInputChange,
+  handleInputFocus,
+  handleInputBlur,
+  handleSuggestionClick,
+}) => {
+  return (
+    <div className={`dropdown ${isFocused
+      ? 'is-active'
+      : ''}`}
+    >
+      <div className="dropdown-trigger">
+        <input
+          type="text"
+          placeholder="Enter a part of the name"
+          className="input"
+          value={inputValue}
+          onChange={handleInputChange}
+          onFocus={handleInputFocus}
+          onBlur={handleInputBlur}
+        />
+      </div>
+
+      <div className="dropdown-menu" role="menu">
+        {showSuggestions && (
+          filteredPeople.length > 0
```

Comment:
```suggestion
          filteredPeople.length
```

can be more simplify